src/Pimple/Container.php

```diff
@@ -92,6 +93,20 @@ public function offsetSet($id, $value)
      */
     public function offsetGet($id)
     {
+        if (isset($this->aliases[$id])) {
+            try {
+                return $this->offsetGet($this->aliases[$id]);
+            } catch (\InvalidArgumentException $e) {
+                throw new \InvalidArgumentException(
+                    sprintf(
+                        'Tryied to get the aliased service "%s" but its identifier "%s" was not defined.',
```

Comment:
the error might not be that the identifier was not defined. It would be better to say that an error occurred, and linking the original exception as the previous exception, to have the full error (btw, it could be an exception coming from elsewhere than Pimple itself)